cf-check/cf-check.c

```diff
@@ -18,6 +19,7 @@ static void print_help()
         "Commands:\n"
         "\tdump - Print the contents of a database file\n"
         "\tdiagnose - Assess the health of one or more database files\n"
+        "\tbackup - Copy database files to a timestamped folder\n"
         "\tversion - Print version information\n"
```

Comment:
Should we not allow an arg of file glob like `/var/cfengine/state/*.lmdb` and instead just use that path always since we mention "database files"?

Reply:
@craigcomstock In my example the glob is expanded by the shell, not cf-check. I don't really want to add wildcard logic, but I plan to make default behavior same as this if you don't specify any files. Not a high priority currently, as I think it's quite easy to workaround. Currently all commands follow the same usage / argument pattern (list of filenames after command), which is nice, IMO.

Reply:
Oh. I get it now. dump/diagnose both operate on a file glob given by the user. Cool. Maybe add `/var/cfengine/state/*.lmdb` as an example somewhere in the --help?

Reply:
It is an example in help menu (the example uses diagnose, but usage is the same). I'm not adding it for backup, because I think it is not such an important command. (Examples for dump, diagnose and repair are appropriate, imo.)